README.md

```diff
@@ -4,247 +4,148 @@ This backend is currently in development and is not advised for Production workf
 
+### Start the Server
+To verify that you installed Django MongoDB Backend and correctly configured your project, run the following command from your project root:
 ```bash
-$ python manage.py startapp myapp --template https://github.com/mongodb-labs/django-mongodb-app/archive/refs/heads/5.0.x.zip
+python manage.py runserver
 ```
-(where "5.0" matches the version of Django that you're using.)
+Then, visit http://127.0.0.1:8000/. This page displays a "Congratulations!" message and an image of a rocket.
 
-### Configuring the `DATABASES` setting
 
-After you've set up a project, configure Django's `DATABASES` setting similar
-to this:
+## Capabilities of Django Backend for MongoDB
 
-```python
-DATABASES = {
-    "default": {
-        "ENGINE": "django_mongodb_backend",
-        "HOST": "mongodb+srv://cluster0.example.mongodb.net",
-        "NAME": "my_database",
-        "USER": "my_user",
-        "PASSWORD": "my_password",
-        "PORT": 27017,
-        "OPTIONS": {
-            # Example:
-            "retryWrites": "true",
-            "w": "majority",
-            "tls": "false",
-        },
-    },
-}
-```
+- **Database Connectivity**  
+    
+  - Directly tune MongoDB connection settings within your Django configuration\!  
+  - Work against a persisted cloud instance of MongoDB for free\!
 
-For a localhost configuration, you can omit `HOST` or specify
-`"HOST": "localhost"`.
 
-`HOST` only needs a scheme prefix for SRV connections (`mongodb+srv://`). A
-`mongodb://` prefix is never required.
+- **Model MongoDB Documents Through Django’s ORM**  
+    
+  - Translate Django model instances to MongoDB documents.  
+  - Create new collections corresponding to models.  
+  - Supports field validation, data storage, updating, and deletion.  
+  - Maps Django's built-in fields to MongoDB data types.  
+  - Provides new custom fields for arrays (ArrayField) and embedded documents (EmbeddedModelField).  
+  - Supports core migration functionalities including creating, deleting, and updating indexes and collections
 
-`OPTIONS` is an optional dictionary of parameters that will be passed to
-[`MongoClient`](https://pymongo.readthedocs.io/en/stable/api/pymongo/mongo_client.html).
 
-`USER`, `PASSWORD`, and `PORT` (if 27017) may also be optional.
+- **Index Management**  
+    
+  - Create single, compound, and unique indexes using Django Indexes  
+  - Create MongoDB partial indexes in Django using Q notation.
 
-For a replica set or sharded cluster where you have multiple hosts, include
-all of them in `HOST`, e.g.
-`"mongodb://mongos0.example.com:27017,mongos1.example.com:27017"`.
 
-Alternatively, if you prefer to simply paste in a MongoDB URI rather than parse
-it into the format above, you can use:
+- **Querying Data**  
+    
+  - Querying API powered through the amazing MongoDB Aggregation Pipeline  
+  - Supports most functions of the Django QuerySet API  
+  - Support Query Annotations and common SQL AGGREGATE operators  
```

Comment:
What are "Query Annotations"?  What does it mean to support SQL aggregate operators on MongoDB? I guess you mean QuerySet.annotate() and QuerySet.aggregate()... you already said most of the QuerySet API is supported....